pkg/model_registry/bentoml.go

```diff
@@ -0,0 +1,132 @@
+package model_registry
```

Comment:
move this to an internal bentoml folder may maybe better for future re-use

Reply:
and we'd better name this kind of model registry to `file-based`, which will not be bound to the internal implementation.